File: BOJ_1330/BOJ_1330.py
import sys
A, B = map(int, sys.stdin.readline().split())
A = int(A)
B = int(B)
if(A>B):
    print(">")
elif(A<B):
    print("<")
elif(A==B):
    print("==")
else:
    print("Wrong")
    
    

File: BOJ_1330/BOJ_1330.cs
using System;

namespace compare{
        public class comparecondition{
                static void Main(){
                        var line = Console.ReadLine();
                        var data = line.Split(' ');
                        var A = int.Parse(data[0]);
                        var B = int.Parse(data[1]);
                        if(A>B){
                                Console.WriteLine(">");
                        }
                        else if(A<B){
                                Console.WriteLine("<");
                        }
                        else if(A==B){
                                Console.WriteLine("==");
                        }
                        else{
                                Console.WriteLine("ERR");
                        }
                }
        }
}
